logging.js

```diff
@@ -157,9 +158,12 @@ export class LogBuilder {
 }
 
 export class LoggingClient {
-	constructor(appId, logger) {
+	constructor(appId, opts, logger) {
```

Comment:
Nit: since options are optional, I might suggest making them the last parameter -- that way you don't need to pass `undefined` in the tests.

Reply:
Yeah, I was thinking it looked nicer to keep the order the same as in the `createClient` function and only the tests would really suffer, but it does make things clunkier in the tests for no reason

Reply:
Well... if you think of it as the options always being the last parameter, then it is consistent. :)